src/libraries/System.Private.CoreLib/src/System/IO/File.cs

```diff
@@ -689,6 +689,63 @@ public static void WriteAllBytes(string path, byte[] bytes)
             RandomAccess.WriteAtOffset(sfh, bytes, 0);
         }
 
+        /// <summary>
+        /// Appends the specified byte array to the end of the file at the given path. 
+        /// If the file doesn't exist, this method creates a new file.
+        /// </summary>
+        /// <param name="path">The path of the file to which the bytes should be appended.</param>
+        /// <param name="bytes">The byte array to append to the file.</param>
+        /// <exception cref="T:System.ArgumentException">
+        /// Thrown when <paramref name="path" /> is a zero-length string, contains invalid characters 
+        /// (which can be queried using the <see cref="M:System.IO.Path.GetInvalidPathChars" /> method), 
+        /// or when <paramref name="bytes" /> is null or empty.
```

Comment:
```suggestion
        /// <paramref name="path" /> is a zero-length string, contains invalid characters
        /// (which can be queried using the <see cref="M:System.IO.Path.GetInvalidPathChars" /> method),
        /// or when <paramref name="bytes" /> is null or empty.
```

Reply:
> when <paramref name="bytes" /> is null or empty

We should not throw when `bytes` is empty. I think this is just a documentation error, no?

Reply:
Indeed, I need to correct the documentation. The behaviour is to throw an exception for a null param, not an empty byte array.